pkgs/development/libraries/qt-5/5.8/qtbase/default.nix

```diff
@@ -266,7 +307,7 @@ stdenv.mkDerivation {
     description = "A cross-platform application framework for C++";
     license = with licenses; [ fdl13 gpl2 lgpl21 lgpl3 ];
     maintainers = with maintainers; [ bbenoist qknight ttuegel ];
```

Comment:
Wouldn't you like to ensure being notified of future proposed changes to `qtbase` by listing yourself as a maintainer?